Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -260,6 +260,13 @@ final class NoShippingPledgeViewController: UIViewController,
           .configureWith(rewardsData: rewardsData, bonusAmount: bonusAmount, pledgeData: pledgeData)
       }
 
+    self.viewModel.outputs.showPledgeOverTimeUI
+      .observeForUI()
+      .observeValues { value in
+        // TODO: Use this flag to hide or show the Pledge Over Time UI [MBL-1814](https://kickstarter.atlassian.net/browse/MBL-1814)
```

Comment:
Nit: Word "flag" here is misleading - we only want to show the UI if the feature flag is on and the project is opted in. Maybe just delete the first part of this sentence; ie `// TODO(MBL-1814): Hide or show pledge over time UI`